test/test_ecs_utils.py

```diff
@@ -17,17 +34,8 @@ def mock_update_service(*, cluster, service, forceNewDeployment):  # noQA - AWS
     return
 
 
-@pytest.mark.parametrize('service_name', [
-    ECSUtils.WSGI, ECSUtils.INDEXER, ECSUtils.INGESTER
-])
-def test_ecs_utils_basic_should_proceed(ecs_utils, service_name):
-    with mock.patch.object(ecs_utils.client, 'update_service', mock_update_service):
-        ecs_utils.update_ecs_service(service_name=service_name)
-
-
-@pytest.mark.parametrize('service_name', [
-    'badservicename', 5, None
-])
-def test_ecs_utils_basic_should_fail(ecs_utils, service_name):
-    with pytest.raises(Exception):
-        ecs_utils.update_ecs_service(service_name=service_name)
+def test_ecs_utils_will_update_services(ecs_utils):
+    """ Tests basic code interaction, no integrated testing right now. """
+    with mock.patch.object(ecs_utils.client, 'list_services', mock_list_ecs_services):
+        with mock.patch.object(ecs_utils.client, 'update_service', mock_update_service):
```

Comment:
If you just don't supply a mock, and instead pick it up using `as`, it will give you a magic mock that you can then ask whether it got called using .assert_called_with(...) to refer to the most recent call or if there are a lot there is a way to get all the calls. e.g., from https://docs.python.org/3/library/unittest.mock.html you can see:
```
mock = Mock(return_value=None)
mock()
mock(3, 4)
mock(key='fish', next='w00t!')
mock.call_args_list
[call(), call(3, 4), call(key='fish', next='w00t!')]
expected = [(), ((3, 4),), ({'key': 'fish', 'next': 'w00t!'},)]
mock.call_args_list == expected
True
```